tests/testsuite/support/mod.rs

```diff
@@ -0,0 +1,14 @@
+macro_rules! t {
```

Comment:
Changed this to `ok_or_panic`, which is the most descriptive thing I could come up with for what this is doing. And using braces since that looks nicer.